doc/nrf/libraries/zigbee/shell.rst

```diff
@@ -384,6 +384,149 @@ Example:
 
 ----
 
+.. _zcl_cmd:
```

Comment:
@mia-ko This documentation was ported from here: https://infocenter.nordicsemi.com/topic/sdk_tz_v4.1.0/zigbee_example_cli_reference.html

Reply:
Thanks for the info, that helped!
I suggested some changes checking against the zcl specification, I hope it looks ok to you.